src/Stripe/Entities/StripeCountrySpec.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Stripe.Infrastructure;
+
+namespace Stripe
+{
+    public class StripeCountrySpec : StripeObject
+    {
+        [JsonProperty("object")]
+        public string Object { get; set; }
+
+        [JsonProperty("livemode")]
+        public bool LiveMode { get; set; }
+
+        [JsonProperty("default_currency")]
+        public string DefaultCurrency { get; set; }
+
+        [JsonProperty("supported_bank_account_currencies")]
+        public Dictionary<string, IEnumerable<string>> SupportedBankAccountCurrencies { get; set; }
+
+        [JsonProperty("supported_payment_currencies")]
+        public IEnumerable<string> SupportedPaymentCurrencies { get; set; }
+
+        [JsonProperty("supported_payment_methods")]
+        public IEnumerable<string> SupportedPaymentMethods { get; set; }
```

Comment:
same with these - I opted for List<> because 1) everything is coming back. there is no delayed execution.   2) there is no need for .ToList() this way.